backend/fms_core/tests/test_resources.py

```diff
@@ -48,9 +49,11 @@ def get_ds():
 CONTAINER_RENAME_CSV = APP_DATA_ROOT / "Container_rename_v3_2_0_B_A_1.csv"
 SAMPLE_EXTRACTION_CSV = APP_DATA_ROOT / "Sample_extraction_v3_3_0_B_A_1.csv"
 SAMPLE_TRANSFER_CSV = APP_DATA_ROOT / "Sample_transfer_v3_2_0_B_A_1.csv"
-SAMPLE_SUBMISSION_CSV = APP_DATA_ROOT / "Sample_submission_v3_2_0_B_A_2.csv"
+SAMPLE_SUBMISSION_CSV = APP_DATA_ROOT / "Sample_submission_v3_4_0_B_A_2.csv"
```

Comment:
This is unlikely to be the second iteration of the sample submission csv for this version since its not out yet ;). Sample_submission_v3_4_0_B_A_2 => Sample_submission_v3_4_0_B_A_1
We increment the counter within each version when we change a test template and the version do not change. We reset the counter when we change the version even if the test template change.